### ISSUE TITLE
Frontendmasters suspended my account after using this tool

### ISSUE THREAD

**Issue by omidpanahi** [20lives/FrontendMastersDownloader#42]
<img width="931" alt="Screen Shot 1400-03-11 at 1 15 19 PM" src="https://user-images.githubusercontent.com/13885292/120294755-c396d300-c2db-11eb-82af-215a15e18241.png">

**Comment by tdias25** [comment]
interesting, did it happened after the first try?

**Comment by omidpanahi** [comment]
Yes, it happened for the first try.

**Comment by techsin** [comment]
same ... my membership ends soon i was too busy to use it while i had it.. im starting to think this repo might be fake

**Comment by 20lives** [comment]
Hi guys, 

This repo works but recently FrontendMasters actively search for users who violate their terms of service.

Please read the [disclaimer](https://github.com/20lives/FrontendMastersDownloader#disclaimer) of this repo.

**Comment by Tanish2409** [comment]
Hi, @omidpanahi did you reply to them as mentioned in the mail and if yes, did your account got restored?

**Comment by techsin** [comment]
honestly not even worth the effort.... there are 100s of books and courses to choose from, most of them are free too. Only thing unique i found on there was the course on streams. Everything else was good but also ubiquitous on internet.

**Comment by omidpanahi** [comment]
Yes @Tanish2409 I replied the mail and  they restored my account  after a couple of days.

**Comment by rolki-png** [comment]
I received suspension too.
Note: The tool failed to auth and got a timeout error before I could manage to download anything.

<img width="728" alt="fem" src="https://user-images.githubusercontent.com/43180177/124471353-0ea17b80-dd9d-11eb-8950-0f2095526416.png">

**Comment by ashishpaul99** [comment]
@omidpanahi brother my account got suspended to please help what to reply to them.

**Comment by omidpanahi** [comment]
@ashishpaul99 
Hi buddy. Since my account was suspended, I apologized and explained what had happened, and let them know I wasn't aware that downloading content via unofficial clients was against their policy. In response, they informed me that my account will be restored within a few days.

**Comment by ashishpaul99** [comment]
@omidpanahi thank you bro for replying, how many days it taken to get back your account.

**Comment by omidpanahi** [comment]
@ashishpaul99 About 5 days.

**Comment by ashishpaul99** [comment]
Thank you brooo.

**Comment by ogeeDeveloper** [comment]
Wow, account suspension...is their any way to hide the usage of scripts such as timeout

**Comment by ibrahimtoru** [comment]
same here. they suspended my account too.

**Comment by Neel278** [comment]
Same goes with me. My account is also suspended after usage of this script.

**Comment by m-naeem66622** [comment]
I think we have to set a bounty for this issue to fix it. :) Currently we have no idea. Hope so there would be a work around to bypass this...

**Comment by winzerprince** [comment]
You can't beat the masters at their own game,  mine got suspended too. I have also noticed their way of handling offline courses on the frontend masters app is pretty unique, never seen anything like it in.
